environment-files/readthedocs.yml

```
@@ -1,5 +1,5 @@
 channels:
-- https://packages.qiime2.org/qiime2/2024.10/amplicon/released
+- https://packages.qiime2.org/qiime2/2025.4/amplicon/passed
 - conda-forge
 - bioconda
 dependencies:
```

```suggestion
dependencies:
  - qiime2-amplicon >= 2025.03.09.04.11.16
```

Try something like this to see if there's a conflict we're just not noticing.